Tests/Brokerages/GDAX/GDAXBrokerageTests.cs

```diff
@@ -0,0 +1,401 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+using Moq;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using QuantConnect.Brokerages;
+using QuantConnect.Brokerages.GDAX;
+using QuantConnect.Interfaces;
+using QuantConnect.Orders;
+using QuantConnect.Packets;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading;
+
+namespace QuantConnect.Tests.Brokerages.GDAX
+{
+
+    [TestFixture()]
+    public class GDAXBrokerageTests
+    {
+
+        #region Declarations
+        GDAXBrokerage _unit;
+        Mock<IWebSocket> _wss = new Mock<IWebSocket>();
+        Mock<IRestClient> _rest = new Mock<IRestClient>();
+        Mock<IAlgorithm> _algo = new Mock<IAlgorithm>();
+        string _orderData;
+        string _orderByIdData;
+        string _openOrderData;
+        string _matchData;
+        string _accountsData;
+        string _holdingData;
+        string _tickerData;
+        Symbol _symbol;
+        const string _brokerId = "d0c5340b-6d6c-49d9-b567-48c4bfca13d2";
```

Comment:
Interesting, what are these?

Reply:
Text files supply sample json to unit test the websockets pipeline abstracted and offline (and part of CI). Pretty much my standard practice. In this case, code coverage is fairly decent.